src/utils/reshapedmsbuild.fs

```diff
@@ -784,7 +792,15 @@ module internal ToolLocationHelper =
 //        instance.GetType().GetPropserty(propName, BindingFlags.Public).SetValue(instance, propValue, null)
 
     type ResolveAssemblyReference () =
-        let assembly = Assembly.Load(new AssemblyName("Microsoft.Build.Tasks.v4.0, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"))
+        let assembly =
+            Assembly.Load(
+                new AssemblyName(
+#if !NETSTANDARD1_5
```

Comment:
AFAIK is this defined automatically when compiling netstandard1.5 with project.json. I have no idea how it works after switching to msbuild. Most likely I should have used another symbolor added it to msbuild but i have no idea...